src/app/components/vertical-icons-component.js

```diff
@@ -206,4 +228,15 @@ const css = csjs`
     bottom: 0;
     right: 0;
   }
+  .statusWithBG
```

Comment:
that one seems not used

Reply:
I know. I have it, for now, we will discuss this in a call. this is the other way to show status. The Circle with the number of errors/warning instead of border and icon. I'll remove either one of them